src/universal/types/schema.flow.js

```diff
@@ -957,8 +957,6 @@ export type SlackIntegration = {
 export type OrgTierEnum = "personal" | "pro" | "enterprise";
 
 export type SuProOrgInfo = {
-  /** The count of active users within the org */
```

Comment:
Oh interesting! Looks like I forgot to rebuild the schema before my last change. Glad this is in here now.